### DISCUSSION TITLE
https://notes.alexkehayias.com/github-codespaces-is-too-slow-and-expensive-to-replace-local-development/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#3] [Announcements]
# https://notes.alexkehayias.com/github-codespaces-is-too-slow-and-expensive-to-replace-local-development/

After trying out GitHub Codespaces for work, it’s clear that—while impressive—it is not a suitable replacement to developing locally.

https://notes.alexkehayias.com/github-codespaces-is-too-slow-and-expensive-to-replace-local-development/

**Comment by markgoho** [comment]
> At time of writing, a CORS error will occur

It would be cool if there were an obvious way to figure out when this article was published

**Reply by alexkehayias** [reply]
@markgoho Published on 2022-05-04

**Comment by magzim21** [comment]
I overcame this limitation of accessing docker commands with
	"features": {
		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {}
     }

**Reply by alexkehayias** [reply]
Oh neat! Looks like this has more info:
https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker-compose
